From edc0d9f8e847ab73e19b313d1c8b9e7d8a06f188 Mon Sep 17 00:00:00 2001
From: Aggelos Tselios <aggelostselios777@gmail.com>
Date: Wed, 24 Dec 2025 13:23:48 +0200
Subject: [PATCH 5/7] os: client: rename get_argmax_fptr to
 get_argmax_from_kern

the _fptr suffix is used in other parts when they replaced apple blocks
functionality. However in this case a more descriptive name is better
(since this function can technically be reused in other places in the
future)

Signed-off-by: Aggelos Tselios <aggelostselios777@gmail.com>
---
 os/client.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/os/client.c b/os/client.c
index efd44a4a7..1ef6c08ae 100644
--- a/os/client.c
+++ b/os/client.c
@@ -124,7 +124,7 @@ DetermineClientPid(struct _Client * client)
 
 #ifdef __APPLE__ /* only required on macOS */
 static void
-get_argmax_fptr(void *arg)
+get_argmax_from_kern(void *arg)
 {
     int *argmax = arg;
     int mib[2];
@@ -183,7 +183,7 @@ DetermineClientCmd(pid_t pid, const char **cmdname, const char **cmdargs)
     {
         static dispatch_once_t once;
         static int argmax;
-        dispatch_once_f(&once, &argmax, get_argmax_fptr);
+        dispatch_once_f(&once, &argmax, get_argmax_from_kern);
 
         int mib[3];
         size_t len = argmax;
-- 
2.52.0

